cli/flox/src/utils/search.rs

```diff
@@ -80,3 +80,17 @@ pub(crate) fn construct_search_params(
     debug!("search params raw: {:?}", params);
     Ok(params)
 }
+
+/// `search` specific curated terms
+pub(crate) fn suggest_curated_package(input: &str) -> Option<&'static str> {
```

Comment:
```suggestion
pub(crate) fn suggest_curated_search_package(input: &str) -> Option<&'static str> {
```
Suggestion: add some specificity since there's another function with exactly the same name

Reply:
it's already in a separate module for `search`, i thought adding another "search" would be a bit redundant..